x-pack/plugins/lens/public/shared_components/coloring/palette_panel_container.tsx

```diff
@@ -5,105 +5,45 @@
-      <EuiFocusTrap disabled={!focusTrapIsEnabled} clickOutsideDisables={true}>
-        <EuiOutsideClickDetector onOutsideClick={closeFlyout} isDisabled={!isOpen}>
-          <div
-            role="dialog"
-            aria-labelledby="lnsPalettePanelContainerTitle"
-            data-test-subj="lns-indexPattern-PalettePanelContainer"
-            className="lnsPalettePanelContainer"
-          >
-            <EuiFlyoutHeader hasBorder className="lnsPalettePanelContainer__header">
-              <EuiFlexGroup gutterSize="s" alignItems="center">
-                <EuiFlexItem grow={false}>
-                  <EuiButtonIcon
-                    color="text"
-                    data-test-subj="lns-indexPattern-PalettePanelContainerBack"
-                    className="lnsPalettePanelContainer__backIcon"
-                    onClick={closeFlyout}
-                    iconType="sortLeft"
-                    aria-label={i18n.translate('xpack.lens.table.palettePanelContainer.back', {
-                      defaultMessage: 'Back',
-                    })}
-                  />
-                </EuiFlexItem>
-
-                <EuiFlexItem>
-                  <EuiTitle size="xs">
-                    <h3
-                      id="lnsPalettePanelContainerTitle"
-                      className="lnsPalettePanelContainer__headerTitle"
-                    >
-                      {title}
-                    </h3>
-                  </EuiTitle>
-                </EuiFlexItem>
-              </EuiFlexGroup>
-            </EuiFlyoutHeader>
-
-            {children && <div className="lnsPalettePanelContainer__content">{children}</div>}
-
-            <EuiFlyoutFooter className="lnsPalettePanelContainer__footer">
-              <EuiButtonEmpty flush="left" size="s" iconType="sortLeft" onClick={closeFlyout}>
-                {i18n.translate('xpack.lens.table.palettePanelContainer.back', {
-                  defaultMessage: 'Back',
-                })}
-              </EuiButtonEmpty>
-            </EuiFlyoutFooter>
-          </div>
-        </EuiOutsideClickDetector>
-      </EuiFocusTrap>
-    </EuiPortal>
-  ) : null;
+  return (
+    <SettingWithSiblingFlyout
+      {...props}
+      SettingTrigger={({ onClick }: { onClick: () => void }) => (
+        <>
+          <EuiFlexItem>
+            <EuiColorPaletteDisplay
+              data-test-subj="lns_dynamicColoring_edit"
+              palette={props.palette}
+              type={FIXED_PROGRESSION}
+              onClick={onClick}
+            />
+          </EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <EuiButtonIcon
+              data-test-subj="lns_colorEditing_trigger"
+              aria-label={i18n.translate('xpack.lens.colorMapping.editColorMappingButton', {
+                defaultMessage: 'Edit palette',
+              })}
+              iconType="controlsHorizontal"
+              onClick={onClick}
+              size="xs"
+            />
```

Comment:
Would it be possible to add a tooltip to this button and have it's text (and the existing `aria-label`) be "Edit colors"?

Reply:
Done!